### PR TITLE
Cumulus Tutorial Refactor & Update

### LINKED ISSUE #133: Update & Test 09-cumulus-workshop
uses non-monthly tag & https://github.com/substrate-developer-hub/substrate-parachain-template/
- [x] remove formatting files (prettier, editorconfig)
- [x] https://github.com/substrate-developer-hub/substrate-parachain-template/pull/73
- [x] error check in content
- [ ] port updates from original (https://github.com/substrate-developer-hub/cumulus-workshop/pull/104)
- [x] Needs cleaning up & fix of links / URLs throughout
- [x] PoV is mentioned for the first time without any context (see [here](https://docs.substrate.io/tutorials/v3/cumulus-workshop/pt2#3-cross-chain-message-passing-xcmp))
- [x] Its confusing what's happening in `/v3/cumulus-workshop/pt3#kickstart`
- [x] [This step](https://docs.substrate.io/tutorials/v3/cumulus-workshop/pt3#2-using-the-template) and the one after seem incomplete or just don't have much of a purpose
- [ ] [Here](https://docs.substrate.io/tutorials/v3/cumulus-workshop/pt3#1-parathread-registration) you loose readers - mean to learn how to register a parathread here? Ah you have multiple steps that follow, I see. I think you should better introduce this section, turning steps 1 and 2 into more of the "overview" of the next steps
- [x] A congrats and next steps section like the other tutorials.
- [x] ~Port Chinese translation & acknowledgement~ new translation needed based on content changes 
- [ ] test polkadot-launch (https://github.com/paritytech/polkadot-launch/pull/143  && https://github.com/substrate-developer-hub/cumulus-workshop/pull/101#issuecomment-935376451 ) 
- [ ] port over all issues or fix
  - [x] https://github.com/substrate-developer-hub/cumulus-workshop/issues/93

### PR DESCRIPTION
**Pull request by NukeManDan**
closes #133 

- [x] #231 bug fix

### CODE REVIEW

**Comment by NukeManDan** [comment]
@jimmychu0807 let's integrate updates here for polkadot launch too :pray: (per https://github.com/substrate-developer-hub/cumulus-workshop/pull/101) 

please make a PR into this PR's branch

File: src/components/DevNavMenu.tsx [hunk 1]
```diff
@@ -685,22 +685,26 @@ DevNavMenuTuts.set('inkWorkshop', [
...
       {
-        title: `Launch a Development Environment`,
-        link: '/tutorials/v3/cumulus-workshop/pt3',
+        title: `Launch a parachain testnet`,
+        link: '/tutorials/v3/cumulus/pt3',
       },
+      //   {
+      //     title: `Register on Rococo`,
+      //     link: '/tutorials/v3/cumulus/pt4',
+      //   },
```
Comment (NukeManDan) [inline src/components/DevNavMenu.tsx:707]: I explicitly want to build this page, but don't want it in the nav bar (yet) as rococo isn't accepting community registrations right now.

File: src/pages/tutorials/v3.tsx [hunk 2]
```diff
@@ -116,14 +114,14 @@ export default function tutorials({ data }: any) {
...
+            title={`Create a Parachain with Cumulus`}
             image={data.tutorialNine}
-            description={`A workshop to configure a Substrate node as a Parachain`}
+            description={`Learn to use the parachain template in local testnet`}
             time={`3 Hours`}
             difficulty={3}
             prerequisites={true}
-            version={`V3.0.0`}
-            link={`/tutorials/v3/cumulus-workshop/pt1`}
+            version={`polkadot-0.9.10`}
```
Comment (NukeManDan) [inline src/pages/tutorials/v3.tsx:123]: using this version throughout for the workshop. it's a PITA to keep track of, as we change this on ~a weekly basis. Would be great if it was more confined to one location to specify.

**Comment by imadarai** [comment]
> images in `tutorials/v3/cumulus/pt4` are broken (a new page added to this tutorial), and I can't find out what I am doing wrong - @imadarai ? Also I think around [this commit](https://github.com/substrate-developer-hub/substrate-docs/pull/203/commits/8288e7a483d0285967cb1ca54193465390f07a9c) I started seeing: image

If everything is hooked up correctly and images are still not showing up, just refresh the dev server with `yarn clean && yarn dev`
This particular error is from Github API, and you can just ignore it by closing the window.  I can also add a patch so it doesn't bother you much in development mode.

File: v3/tutorials/09-cumulus/c-polkadot-launch/index.mdx [hunk 3]
```diff
@@ -0,0 +1,344 @@
+---
+title: Launch a Parachain Testnet
+slug: /tutorials/v3/cumulus/pt3
+version: 'polkadot-0.9.10'
+section: tutorials
+category: parachains
+duration: 3 Hours
+difficulty: 3
+keywords: cumulus, relay chain, parachain, parathread, paraid, template, rococo, xcm, xcmp, collator
```
Comment (NukeManDan) [inline v3/tutorials/09-cumulus/c-polkadot-launch/index.mdx:9]:
```suggestion
keywords:
  [
    cumulus,
    tutorial,
    workshop,
    parachain,
    parathread,
    paraid,
    register,
    template,
    rococo,
    parathread,
    xcm,
    xcmp,
    relay chain,
    collator,
  ]
```
 cc @jimmychu0807 you reverted this change.
This works on my end, and IMHO the more keywords the better for the search engine to pick this up in context, right?

**Comment by NukeManDan** [comment]
There are still issues I will address in #179 to patch this up, but it's in a good state to merge and make incremental small changes moving forward :tada:

**Pull request merged**